ecosystem/indexer-server/typescript/prisma/schema.prisma

```diff
@@ -98,3 +99,68 @@ model write_set_changes {
   @@id([transaction_hash, hash])
   @@index([transaction_hash, address, type], map: "write_set_changes_tx_hash_addr_type_index")
 }
+
+model collections {
+  creator     String   @db.VarChar
+  name        String   @db.VarChar
+  description String?  @db.VarChar
+  maxamount   BigInt?
+  uri         String?  @db.VarChar
+  created_at  DateTime @db.Timestamp(6)
+  inserted_at DateTime @default(now()) @db.Timestamp(6)
+
+  @@id([creator, name])
+}
+
+model ownerships {
+  token_id    String   @db.VarChar
+  owner       String   @db.VarChar
+  amount      Decimal? @db.Decimal
+  updated_at  DateTime @db.Timestamp(6)
+  inserted_at DateTime @default(now()) @db.Timestamp(6)
+
+  @@id([token_id, owner])
+}
+
+model token_activities {
+  event_key        String       @db.VarChar
+  sequence_number  BigInt
+  account          String       @db.VarChar
+  token_id         String?      @db.VarChar
+  event_type       String?      @db.VarChar
+  amount           Decimal?     @db.Decimal
+  created_at       DateTime     @db.Timestamp(6)
+  inserted_at      DateTime     @default(now()) @db.Timestamp(6)
+  transaction_hash String       @db.VarChar(255)
+  transactions     transactions @relation(fields: [transaction_hash], references: [hash], onDelete: NoAction, onUpdate: NoAction, map: "fk_transactions")
+
+  @@id([event_key, sequence_number])
+}
+
+model tokens {
+  token_id    String   @id @db.VarChar
+  creator     String   @db.VarChar
+  collection  String   @db.VarChar
+  name        String   @db.VarChar
+  description String   @db.VarChar
+  max_amount  BigInt?
+  supply      BigInt
+  uri         String   @db.VarChar
+  minted_at   DateTime @db.Timestamp(6)
+  inserted_at DateTime @default(now()) @db.Timestamp(6)
+}
+
+model metadatas {
```

Comment:
metadata plural is shockingly metadata

Reply:
This is likely from the bad naming metadatas as the db name. metadata is used for representing a single record of metadata in DB. To make diesel working, I have to use a different name for the table which is a plural of metadata. Considering options like metadata_records and metadatas, I used a grammatically incorrect metadatas as the name. What name do you recommend in this case?

Reply:
Just `metadata` is fine i think. Yeah like @areshand  said, i pull the schema from postgres directly so it will need to be updated in the diesel `schema.rs`

Reply:
That makes sense :)